docs/reference/commandline/service_create.md

```diff
@@ -136,6 +136,130 @@ $ docker service create \
+
+
+The following options can only be used for named volumes (`type=volume`);
+
+| Option                | Description
+|:----------------------|:--------------------------------------------------------------------------------------------------------------------
+| **volume-driver**     | Name of the volume-driver plugin to use for the volume. If this option is not provided, the default (`"local"`) volume driver is used to create the volume if it does not exist.
+| **volume-label**      | Custom metadata ("labels") to apply to the volume upon creation. Labels are provided as comma-separated list of key/value pairs, for example, `volume-label=hello=world`. For more information about labels, refer to [apply custom metadata](../../userguide/labels-custom-metadata.md).
+| **volume-nocopy**     | Disables propagating the volume by copying files from the container's filesystem to the volume. By default, if an empty volume is attached to a container, the volume is propagated with the files and directories that are present at the mount-path (`dst`) inside the container. Setting this option disables that behavior, and mounts the volume, even if it's empty. This option takes an optional value, and can be either `true` (or `1`), or `false` (or `0`). If this option is provided without a value, then `true` is used as a default (`volume-nocopy=true` is equivalent to `volume-nocopy`).<br /><br />Note that docker does *not* copy files from the container's filesystem for bind-mounts (`type=bind`). Bind-mounts always use the files/directories from the host.
+| **volume-opt**        | Volume driver-specific options to use when creating the volume. Options are provided as comma-separated list of key/value pairs, for example, `volume-opt=some-option=some-value,some-other-option=some-other-value`. For available options, refer to the documentation of the volume driver that is used.
+
+#### Differences between "--mount" and "--volume"
+
+The `--mount` flag features most options that are supported by the `-v` /
+`--volume` flag for `docker run`. There are some differences;
+
+- The `--mount` flag allows specifying a volume driver, and volume driver
+  options *per volume*, without having to create volumes in advance. When using
+  `docker run`, only a single volume driver can be specified (using the
+  `--volume-driver` flag), which is shared by all volumes.
+- The `--mount` flag allows specifying custom metadata ("labels") for the volume,
+  without having to create the volume out of band.
+- When using `type=bind`, the host-path must refer to an *existing* path on the
+  host, and is not automatically created if the path does not exist. If the
+  specified path does not exist on the host, an error is produced, and the
+  service will fail to be deployed succesfully.
+- The `--mount` flag does not allow you to relabel volumes with `Z` or `z`
+
+#### Important considerations when using volumes and bind-mounts
+
+
+
+### Examples
```

Comment:
I'll have a look at the overall structure; was focused on my new content, so I'll change if needed